documentation/docs/student/quiz-training-mode.mdx

```diff
@@ -0,0 +1,200 @@
+import leaderboard from "./assets/quiz-training-mode/leaderboard.png";
+import userSettings from "./assets/quiz-training-mode/user-settings.png";
+import userLeaderboardSettings from "./assets/quiz-training-mode/user-leaderboard-settings.png";
+import Callout from "../../src/components/Callout/Callout";
+import {CalloutVariant} from "../../src/components/Callout/Callout.types";
+import Image, {ImageSize} from "../../src/components/Image/Image";
+
+# Quiz Training Mode
+
+The Quiz Training Mode allows you to reinforce your knowledge through repeated practice of quiz questions. The system uses a spaced-repetition algorithm to ensure you review questions at optimal intervals.
+
+---
+
+## Getting Started
+
+<Image src={trainingTab} alt={"Description"} size={ImageSize.small}></Image>
+
+When you first visit the Quiz Training, a welcome dialog appears explaining the key features:
+
+### How It Works
+
+Quiz Training is based on the principle of spaced repetition. Questions you answer correctly are repeated at longer intervals, while incorrectly answered questions appear more frequently.
+
+### League System
+
+The training uses a motivating league system with five levels:
+
+- **Bronze League:** 0-50 points
+- **Silver League:** 50-150 points
+- **Gold League:** 150-300 points
+- **Diamond League:** 300-500 points
+- **Master League:** 500+ points
+
+Each correctly answered question earns you points, allowing you to advance to higher leagues.
+
+### Scoring
+
+Point allocation depends on the difficulty and your performance on the question. Graded questions count toward your progress, while ungraded questions are only for practice.
+
+### Leaderboard Settings
+
+You can choose whether your name appears in the course leaderboard. You can change this setting at any time.
+
+<Image src={description} alt={"Description"} size={ImageSize.medium}></Image>
+
+---
+
+## Overview Page
+
+After the initial setup, you'll see the training overview with several sections:
+
+### Score Card
+
+The left side shows your current status:
+
+- **League Badge:** Your current league with a progress bar to the next league
+- **Next Questions Due In:** Countdown until the next questions are planned for training according to the spaced-repetition algorithm
+- **Total Questions:** Number of questions in the training pool
+- **Correct Answers:** Total number of correct answers
+- **Wrong Answers:** Total number of wrong answers
+
+<Image src={scorecard} alt={"Scorecard"} size={ImageSize.small}></Image>
+
+### Leaderboard
+
+The right side shows the course leaderboard (if enabled):
+
+- **Your Position:** Highlighted with your rank, name, and score
+- **Top Participants:** Leaderboard of other course participants
+- **Profile Pictures:** Visual representation of participants
+
+<Image src={leaderboard} alt={"Leaderboard"} size={ImageSize.large}></Image>
+
+---
+
+## Starting Training
+
+Click **"Start Training"** to begin practicing.
```

Comment:
Is this a button? if yes I think it would be nice to include a picture of the button (or even better, make a component here that resembles that button). I think that was a nice approach that we had in the previous docu there